Sprint-1/explore/objects.md

```diff
@@ -5,12 +5,17 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+- I get : f log() {[native node]} and when I use typeof(console.log) I get : function
 
 Now enter just `console` in the Console, what output do you get back?
-
+- I get a variety of functions that go with console object
 Try also entering `typeof console`
+- I get that console is an object
 
 Answer the following questions:
 
 What does `console` store?
+- Console stores values/variables that users input / pass to the console 
```

Comment:
The description is not quite correct. Would you ask ChatGPT for a second opinion, or to lookup MDN reference to get a more accurate description?